content/code-security/security-advisories/publishing-a-security-advisory.md

```diff
@@ -92,6 +76,22 @@ Publishing a security advisory deletes the temporary private fork for the securi
 
 {% data reusables.repositories.github-reviews-security-advisories %}
 
+## Requesting a CVE identification number (Optional)
+
+Github can provide a CVE number if needed and anyone with admin permissions to a security advisory can request a CVE identification number for the security advisory.
```

Comment:
If the goal is to emphasize the CVE being optional, then I think it might also be worth mentioning that our CNA should only be assigning to software not covered by other CNAs.


```suggestion
Github can provide a CVE number if needed and the software is not under the scope of another CVE Numbering Authority (CNA). Anyone with admin permissions to a security advisory can request a CVE identification number for the security advisory. If the advisory already has a CVE number, then that can be input in the advisory form.
```